AaxDecrypter/AaxcDownloadConverter.cs

```diff
@@ -67,25 +67,40 @@ protected override bool Step2_DownloadAudiobook()
 
             aaxFile.SetDecryptionKey(downloadLicense.AudibleKey, downloadLicense.AudibleIV);
 
+            aaxFile.ConversionProgressUpdate += AaxFile_ConversionProgressUpdate;
+            //var decryptionResult = OutputFormat == OutputFormat.Mp4a ? aaxFile.ConvertToMp4a(outFile, downloadLicense.ChapterInfo) : aaxFile.ConvertToMp3(outFile);
 
-            if (File.Exists(outputFileName))
-                FileExt.SafeDelete(outputFileName);
-
-            var outputFile =  File.Open(outputFileName, FileMode.OpenOrCreate, FileAccess.ReadWrite);
+            var chapterCount = 0;
+            //aaxFile.ConvertToMultiMp4a(downloadLicense.ChapterInfo, newSplitCallback =>
+            aaxFile.ConvertToMultiMp3(downloadLicense.ChapterInfo, newSplitCallback =>
+            {
+                chapterCount++;
+                var fileName = Path.ChangeExtension(outputFileName, $"{chapterCount}.mp3");
+                if (File.Exists(fileName))
+                    FileExt.SafeDelete(fileName);
+                newSplitCallback.OutputFile = File.Open(fileName, FileMode.OpenOrCreate);
```

Comment:
You do need to give the output file write access. I updated the usage example on AAXClean.